src/components/existing-request/existing-request-display.vue

```diff
@@ -781,6 +782,13 @@ export default class ExistingRequestDisplay extends Mixins(
     return this.affiliateOrLogin()
   }
 
+  async amalgamateYourBusiness (): Promise<void> {
+    // safety check
+    if (!this.isNrApprovedOrConditional) return
+
+    return this.affiliateOrLogin()
+  }
```

Comment:
This is a duplicate of incorporateRegisterYourBusiness() above. Why not call the original one on line 317? In fact, why not use the existing event for amalgamations?

I recommend renaming the event and method names to indicate that they're used for incorporations, registrations and amalgamations.